vite.config.ts

```diff
@@ -0,0 +1,28 @@
+import { resolve } from 'path';
+import { defineConfig } from 'vite';
+
+export default defineConfig({
+	build: {
+		rollupOptions: {
+			input: {
+				main: resolve(__dirname, 'index.html'),
+				'disclosure-widget': resolve(
+					__dirname,
```

Comment:
I generally try to avoid using `__dirname` (a commonjs-only property) in a file with `import`/`export`. Can you use import.meta.url and the URL constructor? Here's an example: https://stackoverflow.com/a/70523299/4898045

Totally fine if the answer is "would rather not/it's just for a demo/etc"

Reply:
It's just a demo, so I'll leave this as is for now. Thanks, though! 😉